src/domain/callout/CalloutsPage.tsx

```diff
@@ -52,10 +51,22 @@ const CalloutsPage = ({ entityTypeName, rootUrl }: CalloutsPageProps) => {
                   hubNameId={hubNameId!}
                   challengeNameId={challengeNameId}
                   opportunityNameId={opportunityNameId}
+                  showCreateButton={canCreateCallout}
                 />
               );
             case CalloutType.Canvas:
-              return <CanvasCallout key={callout.id} callout={callout} buildCanvasUrl={buildLinkToCanvas} />;
+              return (
+                <CanvasCallout
+                  key={callout.id}
+                  callout={callout}
+                  loading={loading}
+                  hubNameId={hubNameId!}
+                  challengeNameId={challengeNameId}
+                  opportunityNameId={opportunityNameId}
+                  buildCanvasUrl={buildLinkToCanvas}
+                  showCreateButton={canCreateCallout}
```

Comment:
Creating callouts and creating canvases are bound to different privileges.

Reply:
Merging to dev, to be re-verified after all-hands session